mkdocs.yml

```diff
@@ -0,0 +1,172 @@
+site_name: GradleUp
+site_description: >-
+  GradleUp is an umbrella GitHub organization focusing
+  on simplifying Java and Android development with Gradle.
+  On our menu: Gradle plugins, developer tools and best practices.
+copyright: >-
+  Copyright &copy; 2024 - GradleUp contributors.
+  The content and the license are licensed under CC BY 4.0.
+  Gradle and the Gradlephant logo (the "Gradle Marks") are registered trademarks of Gradle, Inc. and/or its subsidiaries.
+  In the GradleUp organization, "Gradle" typically means "Gradle Build Tool" and does not reference Gradle Inc. and/or its subsidiaries. 
+repo_url: https://github.com/GradleUp/gradleup.github.io
+repo_name: GitHub Repo
+edit_uri: edit/main/
+site_url: https://www.gradleup.com
+
+# Build
+site_dir: _site
+docs_dir: .
+
+nav:
+  - Projects:
+    - All Projects: projects/README.md
+    - Tools for Android:
+      - Auto Manifest: auto-manifest/README.md
+      - Gradle static analysis plugin: static-analysis-plugin/README.md
+      - Gr8: gr8/README.md
+    - Gradle DevX:
+      - Gratatouille: gratatouille/README.md
+      - NMCP (New Maven Central Publishing): nmcp/README.md
```

Comment:
more of a nit:

The readme title reads a bit odd to me: `Nmcp: New Maven Central Publishing (or New Maven Central Portal too!)`. I think we should be mindful of the README files of the various projects as they will end up on the website.

Also status badges/emojis are all in different places.

Reply:
Yes, updating the downstream READMEs is definitely advised